anvio/docs/programs/anvi-display-contigs-stats.md

```diff
@@ -0,0 +1,42 @@
+This program **lets you look at various stats in your %(contigs-db)s(s)**. 
+
+When you run with all default parameters, as so,
+
+{{ codestart }}
+anvi-display-contigs-stats %(contigs-db)s
+{{ codestop }}
+
+It will open the interactive interface. From there, you'll be able to see a breakdown of the genes in your contigs database. This will look something like this: 
+
+![An example of the anvi'o interface for contigs stats](../../images/contigs-stats-interface-example.png)
+
+Let's walk through the stats displayed here: 
+
+At the top of the page are two graphs: 
+* The bars in the top graph represent every integer N and L statistic from 1 to 100. The y-axis is the respective N length and the x-axis is the L length (the exact values can be seen by hovering over each bar). For a definition of the L and N statistics, see below. In other words, if you had sorted your contigs by length (from longest to shortest), and walked through each one, every time you had seen another 1 percent of your total dataset, you would add a bar to the graph showing the number of contigs that you had seen (the L statistic) and the length of the one you were looking at at the moment (the N statistic). 
```

Comment:
Awesome description! The only thing I'd change is that the x-axis isn't the L count, it is merely the integer N or L statistic, or in your words, the percentage of the total dataset accounted for. :)